src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileValidation.java

```diff
@@ -117,6 +118,11 @@ private ValidationOutputData validatePerformanceProfileData(PerformanceProfile p
                 LOGGER.error("Loading saved performance profiles failed: {} ", e.getMessage());
             }
             StringBuilder errorString = new StringBuilder();
+            // check if the performance profile version is deprecated
+            if (performanceProfile.getProfile_version() < KruizeDeploymentInfo.perf_profile_supported_version) {
```

Comment:
This should be specific to the ROS profile right and not a generic one? If I create a new perf profile with version as `1`, it will fail as deprecated whereas it should not

Reply:
As discussed, this is no longer required. Hence, not moving ahead with the changes.